docs/README.md

```diff
@@ -1,7 +1,22 @@
+# DEVELOPER IMPORTANT NOTE
+
+The docs system for Pact involves a lot of generated files that are then picked up from Github by
+`readthedocs.org`. The implications of this are:
+
+- **Most `.rst` files are autogenerated,** at least in the `en` folder. Manual edits to `.rst` files in `en` are usually in the wrong place:
+
+  - For natives, you need to change the Haskell code that produces the function docs in `Pact.Native` and elsewhere.
+
+  - For everything else, chances are you should be editing a `.md` file such as `pact-reference.md`.
+
+- **Readthedocs.org auto-builds on updates to master**. Thus any changes to `.rst` files should generally only accompany
```

Comment:
Apart from merged to master being auto-built, the rest of this information is already mentioned in this README, there shouldn't be a need to repeat it.

Reply:
Well except that devs constantly forget it. We can add something about "see below" but it's easy to miss the warning below. As this is the other thing people constantly get wrong, I wanted to draw attention to it right away.